pkg/minishift/oc/oc.go

```diff
@@ -226,3 +225,23 @@ func mergeKubeConfigs(configs []*clientcmdapi.Config) *clientcmdapi.Config {
 	}
 	return mergedConfig
 }
+
+// getGlobalKubeConfigPath returns the path to the first entry in KUBECONFIG environment variable
+// or if KUBECONFIG not set then ~/home/.kube/config
+func getGlobalKubeConfigPath() (string, error) {
```

Comment:
For what it's worth, why getGlobalKubeConfigPath? What's global about this? Is not a simple (get)KubeConfigPath more fitting?

Reply:
The godocs are still wrong. Either you use $HOME/.kube/config or ~/.kube/config. '~/home' is wrong.